test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -635,22 +635,36 @@ def addSuccess(self, test):
     def _cleanup(self):
         self.logger.info('Cleaning up temporary files')
         du = DshUtils()
-        tmpdir = tempfile.gettempdir()
-        ftd = []
-        if tmpdir:
+        dirlist = set(['/tmp', '/var/tmp'])
+        # get tmp dir from the environment
+        for envname in 'TMPDIR', 'TEMP', 'TMP':
+            dirname = os.getenv(envname)
+            if dirname: 
+                dirlist.add(dirname)
+
+        for tmpdir in dirlist:
+            self.logger.info('Cleaning up ' + tmpdir + ' dir')
+            ftd = []
             files = du.listdir(path=tmpdir)
             bn = os.path.basename
             ftd.extend([f for f in files if bn(f).startswith('PtlPbs')])
             ftd.extend([f for f in files if bn(f).startswith('STDIN')])
+            ftd.extend([f for f in files if bn(f).startswith('pbsscrpt')])
+            ftd.extend([f for f in files if bn(f).startswith('pbs.conf.')])
+            ftd.extend([f for f in files if bn(f).startswith('pbs.')])
```

Comment:
I suggest updating it with a regular expression to check pbs.\d+

Reply:
@anamikau : 'pbs.' will serve the purpose. Do you still prefer regex over string match?

Reply:
@kjakkali the reason I asked you to check that there is a digit to make sure you are deleting jobdir only. There might be other files that one might have created with prefix "pbs." and we should not just be cleaning everything that starts with 'pbs.'.
Do you agree?

Reply:
done